preparing-to-code/ada-languages/README.md

```diff
@@ -1,43 +1,59 @@
 
 ## Classification
+
 There are two main ways we can classify programming languages, by level and by type (compiled or interpreted).
 
 ### By level
+
 The level describes how close the language is to being [binary code](https://en.wikipedia.org/wiki/Binary_code). Low-level languages like C, tend to be more difficult for humans to read than other languages. High-level languages like Java, Python, and Ruby tend to be more abstract, easier for humans to read. High-level languages also do a lot more of the background tasks for the software developer.
 
 ### By compiled or interpreted
+
 Compiled languages are converted to binary code (e.g., C, C++, Java) before they are executed. Interpreted languages can be translated or directly executed, the code does not get directly converted to binary, the program that runs the program is the one that communicates directly with the computer.
 
 ![Compiled vs Interpreted](./images/compiled-interpreted.png)
 
 In Addition, these compiled and interpreted languages can be divided into dynamically typed and statically typed languages. Statically typed programming languages do type checking (the process of verifying and enforcing the constraints of types) at compile-time as opposed to run-time. Dynamically typed programming languages do type checking at run-time as opposed to compile-time.
 
 ## Languages
+
 ### Ruby
-Ruby is a high-level, dynamically typed language.  Ruby is different than other programming languages. It was written with human communication in mind, rather than computer optimization. Its creator wanted to create a programming language that was fun for programmers. Ruby is a programming language which was created to make developers have fun and be productive at the same time. Ruby was first written in 1993 by [Yukihiro "Matz" Matsumoto](https://twitter.com/yukihiro_matz). _"Ruby is for People. I hope to see Ruby help every programmer in the world to be productive, and to enjoy programming, and to be happy. That is the primary purpose of Ruby language."_ - Matz
+
+[Ruby](https://www.ruby-lang.org/en/) is a high-level, dynamically typed language.  Ruby is different than other programming languages. It was written with human communication in mind, rather than computer optimization. Its creator wanted to create a programming language that was fun for programmers. Ruby is a programming language which was created to make developers have fun and be productive at the same time. Ruby was first written in 1993 by [Yukihiro "Matz" Matsumoto](https://twitter.com/yukihiro_matz). _"Ruby is for People. I hope to see Ruby help every programmer in the world to be productive, and to enjoy programming, and to be happy. That is the primary purpose of Ruby language."_ - Matz
```

Comment:
If we're doing an update JavaScript "is a dynamically typed and interpreted programming language" but Ruby "is a high-level, dynamically typed language".